lesson05/script.js

```diff
@@ -0,0 +1,25 @@
+const ul = document.getElementById('js-list');
+const lists = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+   ]
+
+const promise = new Promise(resolve => {
+    resolve(lists)
+});
+
+promise.then(() => {
```

Comment:
IMO
I think you should set the argument here. 
``promise.then((lists) => {...``
This is because we need to receive the resolved values here.
Now, when I turn off the value of resolve``resolve()``, it still works, so maybe you have referenced the object globally. The resolved value is not being used.

Reply:
I made a mistake!
The value received by resolve should be stored in a variable.

I need to write:
`promise.then. ((lists) => {...`